winum.el

```diff
@@ -97,7 +97,9 @@ Example: always assign *Calculator* the number 9 and *NeoTree* the number 0:
 (defcustom winum-auto-setup-mode-line t
   "When nil, `winum-mode' will not display window numbers in the mode-line.
 You might want this to be nil if you use a package that already manages window
-numbers in the mode-line.")
+numbers in the mode-line."
+  :group 'winum
+  :type 'boolean)
```

Comment:
Could you please add a space after the `:type`, to have them aligned like I did for the other `customize` options ? :smile_cat: